oncopipe/oncopipe/__init__.py

```diff
@@ -83,7 +83,6 @@ def set_config(self, config):
 
```

Comment:
I noticed that this file has had a bunch of empty lines removed. Could this be reverted?

Reply:
I don't know how this even happened but I don't see why we can't just roll it back if those are the only changes. How can we figure out the cause of this?